### PR TITLE
Feature/metrics

### PR DESCRIPTION
**Pull request by pavel-collab**
Add some new metrics and tables that help to expand abilities of the application. Pay attention to the new tables (you can read about it in docs) and primary and foreign keys.

### CODE REVIEW

File: db_connection.py [hunk 1]
```diff
@@ -49,24 +51,89 @@ def start_docker_client(self):
...
-                (container_id, cpu_usage, memory_usage)
-            )
-            self.conn.commit()
+                for network, net_stat in network_stats.items():
+                    self.cursor.execute(
+                        "INSERT INTO container_networks (id, network_name, resieved_bytes, transmited_bytes) VALUES (%s, %s, %s, %s)",
+                        (container_id, network, net_stat['resieved_bytes'], net_stat['transmited_bytes'])
+                    )
+                self.conn.commit()
+            except errors.UniqueViolation as e:
```
Comment (barannikovav) [inline db_connection.py:91]: What is idea to catch different types of exceptions, including universal `Exception`, but to process them in a same way?

File: docs/timescaledb.md [hunk 2]
```diff
@@ -76,4 +76,63 @@ ALTER TABLE container_stats
     SET (timescaledb.compress, 
          timescaledb.compress_orderby='time', 
          timescaledb.compress_segmentby = 'container_id');
+```
+
+## Создание дополнительных таблиц
+
+Таблица для общей информации по контейнерам. Содержит имя контейнера, соответствующий id и ограничение по памяти.
+```
+-- Создаем тмаблицу
```
Comment (barannikovav) [inline docs/timescaledb.md:85]: Correct 
> тмаблицу

File: docs/timescaledb.md [hunk 3]
```diff
@@ -76,4 +76,63 @@ ALTER TABLE container_stats
...
+CREATE TABLE container_info (id TEXT NOT NULL, 
+                             name TEXT, 
+                             memory_limit DOUBLE PRECISION);
+
+-- Создаем первычный ключ в этой таблийе (она будет основной, все остальные таблицы будут ссылаться на нее)
```
Comment (barannikovav) [inline docs/timescaledb.md:90]: Correct:
> первычный, таблийе

File: docs/timescaledb.md [hunk 4]
```diff
@@ -76,4 +76,63 @@ ALTER TABLE container_stats
...
+ALTER TABLE container_info ADD CONSTRAINT container_id PRIMARY KEY (id);
+```
+
+Более детальная статистика по занятовсти CPU. Содержит время, занимаемое на CPU в kernel, user и system состояниях.
```
Comment (barannikovav) [inline docs/timescaledb.md:94]: Correct:
> занятовсти

File: docs/timescaledb.md [hunk 5]
```diff
@@ -76,4 +76,63 @@ ALTER TABLE container_stats
...
+```
+-- заводим таблицу
```
Comment (barannikovav) [inline docs/timescaledb.md:96]: From capital letter

File: docs/timescaledb.md [hunk 6]
```diff
@@ -76,4 +76,63 @@ ALTER TABLE container_stats
...
+-- Настраиваем сжатие данных
+ALTER TABLE container_detailed_statistics 
+    SET (timescaledb.compress, 
+         timescaledb.compress_orderby='time', 
+         timescaledb.compress_segmentby = 'id');
+```
+
+Статистика по сетевым соединениям контейнеров.
+```
+-- Создаем тмаблицу
```
Comment (barannikovav) [inline docs/timescaledb.md:119]: Correct
> тмаблицу

**Review by barannikovav — CHANGES_REQUESTED** [review]
Some mistake correction needed. And question about SQL tables architecture: is it a good  practice to have TEXT id? I am not sure, but there are some problems with such implementation, please check it

File: db_connection.py (hunk 1 above)
Comment (pavel-collab) [reply db_connection.py:91]: Resolved. For now it's useless

File: docs/timescaledb.md (hunk 2 above)
Comment (pavel-collab) [reply docs/timescaledb.md:85]: resolved

File: docs/timescaledb.md (hunk 3 above)
Comment (pavel-collab) [reply docs/timescaledb.md:90]: resolved

File: docs/timescaledb.md (hunk 4 above)
Comment (pavel-collab) [reply docs/timescaledb.md:94]: resolved

File: docs/timescaledb.md (hunk 5 above)
Comment (pavel-collab) [reply docs/timescaledb.md:96]: resolved

File: docs/timescaledb.md (hunk 6 above)
Comment (pavel-collab) [reply docs/timescaledb.md:119]: resolved

**Comment by pavel-collab** [comment]
> Some mistake correction needed. And question about SQL tables architecture: is it a good practice to have TEXT id? I am not sure, but there are some problems with such implementation, please check it

I got what do you want to say. I will create a particular Issue for this task. Not in this issue anymore.

**Review by barannikovav — APPROVED** [review]
All my comments have been revised in this version.

**Pull request merged**